src/features/TaskScreen/TaskScreen.tsx

```diff
@@ -86,29 +86,34 @@ function TaskScreen({ route, navigation }: TaskScreenPropTypes) {
 
   const renderScene = (scene: Scene) => {
     switch (scene.route.key) {
-      case Status.ToDo.toString():
-        return <TaskListView tasks={tasks} addFunc={addTaskFunc} />;
-      case Status.InProgress.toString():
+      case State.ToDo.toString():
+        return (
+          <TaskListView
+            tasks={tasks.filter((task: Task) => task.state === State.ToDo)}
+            addFunc={addTaskFunc}
+          />
+        );
+      case State.InProgress.toString():
         return (
           <TaskListView
             tasks={tasks.filter(
-              (task: Task) => task.state === Status.InProgress
+              (task: Task) => task.state === State.InProgress
             )}
-            addFunc={addTaskFunc}
+            addFunc={null}
```

Comment:
Why not enabling addFunc like in ToDo?

Reply:
I think we wanted to only be able to create tasks in the ToDo column like it is implemented in the browser version of the app. 🤔 But we can discuss this again.

Reply:
I think I missed that xD